counterparty-core/counterpartycore/lib/api/util.py

```diff
@@ -11,17 +12,20 @@
 logger = logging.getLogger(config.LOGGER_NAME)
 
 
-def check_last_parsed_block(blockcount):
+def check_last_parsed_block(db, blockcount):
     """Checks database to see if is caught up with backend."""
+    last_block = ledger.get_last_block(db)
```

Comment:
two minutes or one minute? Am I reading this right?

Reply:
(two minutes seems like a lot!)

Reply:
fixed the title and the release notes.